resources/js/tasks/components/TasksList.vue

```diff
@@ -289,8 +289,13 @@ export default {
       </a>`;
     },
     setupColumns() {
-      const columns = this.getColumns();
       this.tableHeaders = this.getColumns();
+      this.tableHeaders.forEach((column) => {
+        if (column.field === "case_title") {
+          // eslint-disable-next-line no-param-reassign
```

Comment:
please remove commented code

Reply:
This line were removed, and are handled in SavedSearchController